internal/awsservice/ecs.go

```diff
@@ -90,41 +78,19 @@ func GetClusterName(clusterArn string) string {
 }
 
 func listContainerInstances(clusterArn string) (*ecs.ListContainerInstancesOutput, error) {
-	svc, ctx, err := getEcsClient()
-	if err != nil {
-		return nil, err
-	}
 
 	input := &ecs.ListContainerInstancesInput{
```

Comment:
I'm not really seeing the benefit here. Why not just use the client directly? How much more difficult is it for someone to write:
```go
client.ListContainerInstances(context.Background(), &ecs.ListContainerInstancesInput{Cluster: aws.String(clusterArn)})
```

Reply:
I'm fine with write it too. Try to keep the code small as much as possible though.

Reply:
I remember discussing something similar in a previous PR. I don't think it's necessary to have these functions if they're not either transforming the input or triggering some other change. We don't need to have this extra layer.

Reply:
The previous PR has a broader change regarding using the `interface` with the SDK. However, the extra layer you mentioned are not in scope of the PR. But I'm fine with changing it.